docs/editors/new-editor.md

```diff
@@ -168,9 +168,7 @@ Possible values:
 
 - `off` (default): don't start a server with the Metals HTTP client.
 - `on`: start a server with the [Metals HTTP client] to interact with the server
-  through a basic web UI. This option is needed for editor clients like Sublime
```

Comment:
Let's add back `This option is needed for editor clients that don't support necessary requests such as
  `window/showMessageRequest`.`